### PR TITLE
[VEN-2497][VEN-2498]: XVS and XVSBridge deployment on Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2497 and VEN-2498

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -110,10 +126,10 @@ export const bridgeConfig: BridgeConfig = {
...
       { method: "setMinDstGas(uint16,uint16,uint256)", args: [10161, 0, "300000"] },
-      { method: "setMaxDailyLimit(uint16,uint256)", args: [10161, "500000000000000000000"] },
-      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000"] },
-      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10161, "500000000000000000000"] },
-      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000"] },
+      { method: "setMaxDailyLimit(uint16,uint256)", args: [10161, "50000000000000000000000"] },
+      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000000"] },
+      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10161, "50000000000000000000000"] },
+      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000000"] },
     ],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:133]: Should we include the invocations setting the limits for opBNB testnet and arbitrum sepolia?

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -128,10 +144,10 @@ export const bridgeConfig: BridgeConfig = {
...
-      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10102, "10000000000000000000"] },
-      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10102, "500000000000000000000"] },
-      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10102, "10000000000000000000"] },
+      { method: "setMaxDailyLimit(uint16,uint256)", args: [10102, "50000000000000000000000"] },
+      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10102, "10000000000000000000000"] },
+      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10102, "50000000000000000000000"] },
+      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10102, "10000000000000000000000"] },
     ],
   },
   ethereum: {
```
Comment (chechu) [inline helpers/deploymentConfig.ts:153]: Should we include the invocations setting the limits for opBNB mainnet? In order to keep updated the configuration

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -110,10 +126,10 @@ export const bridgeConfig: BridgeConfig = {
...
-      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000"] },
-      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10161, "500000000000000000000"] },
-      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000"] },
+      { method: "setMaxDailyLimit(uint16,uint256)", args: [10161, "50000000000000000000000"] },
+      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000000"] },
+      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [10161, "50000000000000000000000"] },
+      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [10161, "10000000000000000000000"] },
     ],
   },
   bscmainnet: {
```
Comment (chechu) [inline helpers/deploymentConfig.ts:135]: Should we add the invocations setting the limits with arbitrum one?

**Review by chechu — DISMISSED** [review]

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: I think we should to keep it's track.
https://github.com/VenusProtocol/token-bridge/pull/25/commits/c16f8f99ca35ff2a6464a04f369ada4b3e3bf5db

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts:143]: https://github.com/VenusProtocol/token-bridge/pull/25/commits/c16f8f99ca35ff2a6464a04f369ada4b3e3bf5db

File: helpers/deploymentConfig.ts (hunk 3 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts:137]: https://github.com/VenusProtocol/token-bridge/pull/25/commits/c16f8f99ca35ff2a6464a04f369ada4b3e3bf5db

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -108,58 +132,28 @@ export const xvsTokenPermissions = ["mint(address,uint256)", "burn(address,uint2
...
-      { method: "setMinDstGas(uint16,uint16,uint256)", args: [102, 0, "300000"] },
-      { method: "setMaxDailyLimit(uint16,uint256)", args: [102, "50000000000000000000000"] },
-      { method: "setMaxSingleTransactionLimit(uint16,uint256)", args: [102, "10000000000000000000000"] },
-      { method: "setMaxDailyReceiveLimit(uint16,uint256)", args: [102, "50000000000000000000000"] },
-      { method: "setMaxSingleReceiveTransactionLimit(uint16,uint256)", args: [102, "10000000000000000000000"] },
-    ],
+    methods: [...createMethodEntries(102), ...createMethodEntries(101), ...createMethodEntries(110)],
+  },
+  arbitrumsepolia: {
+    methods: [...createMethodEntries(10102), ...createMethodEntries(10202), ...createMethodEntries(10231)],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:153]:
```suggestion
    methods: [...createMethodEntries(10102), ...createMethodEntries(10202), ...createMethodEntries(10161)],
```

Comment (Debugger022) [reply helpers/deploymentConfig.ts]: https://github.com/VenusProtocol/token-bridge/pull/25/commits/eddc2b2632383686b6324d494e41bf61b1cc75a7

**Review by chechu — APPROVED** [review]

**Review by chechu — DISMISSED** [review]

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -177,7 +171,31 @@ export async function getPreConfiguredAddresses(networkName: string): Promise<Pr
...
   }
 }
+
+function createMethodEntries(
+  chainId: number,
+  minDstGas: number = 300000,
+  maxDailyLimit: BigNumber = parseUnits("50000", 18),
+  maxSingleTransactionLimit: BigNumber = parseUnits("10000", 18),
+  maxDailyReceiveLimit: BigNumber = parseUnits("50000", 18),
+  maxSingleReceiveTransactionLimit: BigNumber = parseUnits("10000", 18),
```
Comment (chechu) [inline helpers/deploymentConfig.ts:189]:
```suggestion
  maxDailyReceiveLimit: BigNumber = parseUnits("51000", 18),
  maxSingleReceiveTransactionLimit: BigNumber = parseUnits("10200", 18),
```

Comment (Debugger022) [reply helpers/deploymentConfig.ts]: https://github.com/VenusProtocol/token-bridge/pull/25/commits/345f9bb8b6188fee6fcf36464574eb7bd91c1913

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**